project-keeper/src/main/java/com/exasol/projectkeeper/githuboutput/OutputPublisherFactory.java

```diff
@@ -0,0 +1,20 @@
+package com.exasol.projectkeeper.githuboutput;
+
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.Optional;
+
+class OutputPublisherFactory {
+    private final Map<String, String> environment;
+
+    OutputPublisherFactory(final Map<String, String> environment) {
+        this.environment = environment;
+    }
+
+    OutputPublisher create() {
```

Comment:
I assume that you might plan to add more factory methods.
The current method is name create but creates a specific `OutputPublisher`, i.e. `FileOutputPublisher`.
Maybe we should rename the method to `fileOutputPublisher()`?

Reply:
No, this returns a `NullOutputPublisher` when the environment variable is missing.